ci/jenkins/README.md

```diff
@@ -0,0 +1,19 @@
+## Antrea CI: Jenkins
+
+### Reasons for Jenkins
+We have tests as Github Action but Jenkins allows tests running on a cluster of multiple nodes and offers better environment setup options.
+
+### Requirements
+Yaml files under ```ci/jenkins/jobs``` can be generated via jenkins-job-builder. If you want to try out the tests on your local jenkins setup, please notice the following requirements:
+* Jenkins setup
+  * Plugins: ghprb, throttle-concurrents
+* Install [jenkins-job-builder](https://docs.openstack.org/infra/jenkins-job-builder/index.html)
+* Define your ```ANTREA_GIT_CREDENTIAL``` which is the credential for your private repo
```

Comment:
you don't need triple backticks for inline code highlighting, just a single one.

Reply:
Updated.